utils/camping_argparser.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import logging
+import sys
+
+from datetime import datetime
+
+from enums.date_format import DateFormat
+
+
+class CampingArgParser(argparse.ArgumentParser):
```

Comment:
s/ArgParser/ArgumentParser/

Reply:
Can you explain a little more on this comment?

Reply:
Just a nit, as in rename ArgParser to ArgumentParser. My bad, sed terminology 😛

Reply:
Oh hahah. I see now.